icons/life-buoy.json

```
@@ -3,7 +3,11 @@
   "tags": [
     "help",
```

Some other missing stuff:

```suggestion
    "preserver",
    "life belt",
    "lifesaver",
    "help",
```